module/Company/src/Company/Controller/AdminController.php

```diff
@@ -0,0 +1,436 @@
+<?php
+
+namespace Company\Controller;
+
+use Zend\Mvc\Controller\AbstractActionController;
+use Zend\View\Model\ViewModel;
+
+class AdminController extends AbstractActionController
+{
+    /**
+     *
+     * Action that displays the main page
+     *
+     */
+
+    public function indexAction()
+    {
+        // Get useful stuff
+        $companyService = $this->getCompanyService();
+
+        // Initialize the view
+        $vm = new ViewModel(array(
+            'companyList' => $companyService->getHiddenCompanyList(),
+        ));
+
+        return $vm;
+    }
+
+    /**
+     *
+     * Action that allows adding a company
+     *
+     */
+
```

Comment:
PSR-2 is irrelevant when it comes to Docblocks. Because docblocks are specified by `phpDocumentor`, which is a slightly changed variant of javadoc for PHP.